starter-code/index.html

```diff
@@ -2,41 +2,78 @@
 <html lang="en">
 <head>
   <meta charset="UTF-8">
+  <link rel="stylesheet" href="css/main.css">
+  <link href="https://fonts.googleapis.com/css?family=Montserrat:100,200,300,400" rel="stylesheet">
   <title>Spotify</title>
 </head>
 <body>
-Premium Discover Help Download
+  <div class="navwrap">
+  <nav class="navbar">
+    <img class="spotify-logo" src="images/spotify-logo.png">
+    <ul class="nav-ul">
+      <li>Premium</li>
+      <li>Discover</li>
+      <li>Help</li>
+      <li>Download</li>
+    </ul>
+  <nav>
+  </div>
+  <div class="clearfloats"></div>
+  <content>
```

Comment:
Don't use this. It doesn't mean what you think it does. `div` is decent enough here. `section` or `main` are semantic alternatives.